src/functions/user.php

```diff
@@ -21,3 +23,8 @@ function user_is_logged_in(): bool
 {
     return \Drupal::currentUser()->isAuthenticated();
 }
+
+function user_role_load(string $rid): ?RoleInterface
+{
+    return Role::load($rid);
```

Comment:
Can we use long-form `\Drupal::entityTypeManager()->getStorage('user_role')->load($rid);`

Reply:
I suppose so. I just copied code I found in core.

Reply:
For clarity I changed this to use our extended Role class.